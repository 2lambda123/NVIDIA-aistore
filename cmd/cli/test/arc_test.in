echo "0123456789" > $OBJECT_1.txt

ais bucket create ais://$BUCKET_1 | awk '{print $1 " " $2}'
ais bucket create ais://$BUCKET_2 | awk '{print $1 " " $2}'

ais object put $OBJECT_1.txt ais://$BUCKET_1/obj_1.txt
ais object put $OBJECT_1.txt ais://$BUCKET_1/obj_2.txt
ais object put $OBJECT_1.txt ais://$BUCKET_1/obj_3.txt

ais object put --source-bck=ais://$BUCKET_1 --template="obj_{1..3}.txt" --archive ais://$BUCKET_2/tmpl.tar
ais object put --source-bck=ais://$BUCKET_1 --list="obj_1.txt,obj_2.txt,obj_3.txt" --archive ais://$BUCKET_2/list.tar

sleep 5 // IGNORE // wait for all archives are created

ais object put $OBJECT_1.txt ais://$BUCKET_2/tmpl.tar -archpath=fourth.txt

ais ls ais://$BUCKET_2 --prefix=list --archive | wc -l
ais ls ais://$BUCKET_2 --prefix=tmpl --archive | wc -l
