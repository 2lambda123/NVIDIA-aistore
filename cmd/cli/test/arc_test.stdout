"ais://$BUCKET_1" created
"ais://$BUCKET_2" created
PUT "obj_1.txt" to ais://$BUCKET_1
PUT "obj_2.txt" to ais://$BUCKET_1
PUT "obj_3.txt" to ais://$BUCKET_1
^Creat.*$
^Creat.*$
^APPEND .* to object "ais://$BUCKET_2/tmpl.tar\[/fourth.txt\]".*$
5
6
